build-scripts/bin/mathesar.sh

```diff
@@ -221,6 +221,7 @@ run_mathesar() {
   gunicorn_args=(
     config.wsgi
     -b "0.0.0.0:${MATHESAR_PORT}"
+    --workers $(($(nproc) + 1)) --threads $((2 * $(nproc)))
```

Comment:
This is the only relevant change.

Reply:
I'm curious what @mathemancer thinks but I strongly suspect we should expose this configuration to users, through env vars or yet another config file, or both, so they can tune these values for their particular setup.